CHANGES.md

```diff
@@ -10,6 +10,13 @@ Rust image aims to be a pure-Rust implementation of various popular image format
 
 ## Changes
 
+### Version 0.23.10
+
+- Added AVIF encoding capabilities using the `ravif` crate. Please note that
+  the feature targets the stable compiler and is not enabled by default.
```

Comment:
stable -> latest stable

Reply:
Ah yes, forgot that I mentioned that here as well :+1: